jasstests/jassrealtime/core/test_schema_list.py

```diff
@@ -138,6 +269,14 @@ def test_add_json_schema(self):
         schemasInfo = self.schemaList.get_json_schemas_infos()
         self.assertEqual(2, len(schemasInfo))
 
+    def test_add_json_schema_string_array(self):
+        jsonSchema = json.loads(JSON_SCHEMA_WITH_STRING_ARRAY)
+        self.schemaList.add_json_schema(jsonSchema, nestedFields=["offsets"])
+        time.sleep(1)
+        schemasInfo = self.schemaList.get_json_schemas_infos()
+        self.assertEqual(1, len(schemasInfo))
+        # adding a name
+
```

Comment:
Ideally i would add one annotation to make sure that string arrays are effectively working.

Reply:
Good idea

Reply:
In the latest commit.